.github/workflows/lint.yml

```diff
@@ -0,0 +1,17 @@
+name: lint
```

Comment:
It it that useful to have format and lint as separate workflows? Is it just so both run if one fails?

Reply:
Yeah it's so that the checkmarks are really easy to read. One for each thing that could fail. A little wasteful but :shrug: